doc/docs/Python_Tutorials/Mode_Decomposition.md

```diff
@@ -148,13 +148,13 @@ Phase of a Total Internal Reflected Mode
 
 ![](../images/refl_coeff_flat_interface.png#center)
 
-The key item to consider in these types of calculations is the *location of the mode monitor relative to the interface*. The mode monitor is a line extending the entire length of the cell in the $y$ direction. In order to compare the result with the Fresnel equations, the phase of the reflection coefficient must be computed *exactly* at the interface. However, it is problematic to measure the reflection coefficient exactly at the interface because the amplitude of the left-going wave drops discontinuously to zero across the interface. For this reason, the mode monitor must be positioned away from the interface (somewhere within the $n_1$ medium) and the measured phase must be corrected in post processing to account for this offset.
+The key item to consider in these types of calculations is the *location of the mode monitor relative to the interface*. The mode monitor is a line extending the entire length of the cell in the $y$ direction. In order to validate the simulated result using the Fresnel equations, the phase of the reflection coefficient must be computed *exactly* at the interface. However, it is problematic to measure the reflection coefficient exactly at the interface because the amplitude of the left-going wave drops discontinuously to zero across the interface. For this reason, the mode monitor must be positioned away from the interface (somewhere within the $n_1$ medium) and the measured phase must be corrected in post processing to account for this offset.
 
-The calculation of the reflection coefficient requires two separate runs: (1) a normalization run involving just the source medium ($n_1$) to obtain the incident fields, and (2) the main run including the interface whereby the incident fields from (1) are first subtracted from the monitor to obtain only the reflected fields. The mode coefficient in (2) divided by (1) is, by definition, the reflection coefficient.
+The calculation of the reflection coefficient requires two separate runs: (1) a normalization run involving just the source medium ($n_1$) to obtain the incident fields, and (2) the main run including the interface whereby the incident fields from (1) are first subtracted from the monitor to obtain only the reflected fields. The mode coefficient in (2) divided by (1) is, by definition, the reflection coefficient. The phase is the [argument](https://en.wikipedia.org/wiki/Argument_(complex_analysis)) of the complex reflection coefficient. Note: there is a [convention](https://en.wikipedia.org/wiki/Total_internal_reflection#Phase_shifts), followed by Meep, used to define the phase which is based on the *sign* of the mode coefficients, in particular for the $P$ polarization.
```

Comment:
```suggestion
The calculation of the reflection coefficient requires two separate runs: (1) a normalization run involving just the source medium ($n_1$) to obtain the incident fields, and (2) the main run including the interface whereby the incident fields from (1) are first subtracted from the monitor to obtain only the reflected fields. The mode coefficient in (2) divided by (1) is, by definition, the reflection coefficient. The phase is the [argument](https://en.wikipedia.org/wiki/Argument_(complex_analysis)) of the complex reflection coefficient.  Whenever one defines a phase, it is relative to some choice of reference — in particular, the phase of the mode profile, which is chosen so that the electric field is purely real and its sign is chosen following a [typical convention](https://en.wikipedia.org/wiki/Total_internal_reflection#Phase_shifts) used for Fresnel coefficients.
```